scim2/scim2_connector.bal

```diff
@@ -122,8 +122,7 @@ function ScimConnector::getListOfUsers() returns (User[]|error) {
     var res = httpEP->get(SCIM_USER_END_POINT, message = request);
     match res {
         error err => {
-            Error = { message: failedMessage + err.message, cause: err.cause };
-            return Error;
+            return err;
         }
```

Comment:
You can remove the `{` in the match pattern for error since it only contains single line. 
Example: 
```ballerina
error err => return err;
```